stdlib/os/__init__.pyi

```diff
@@ -887,8 +915,11 @@ else:
         ) -> int: ...
 
 if sys.platform != "win32":
-    class sched_param(NamedTuple):
-        sched_priority: int
+    @final
+    class sched_param(structseq[int]):
+        def __new__(cls, sched_priority: int) -> sched_param: ...
```

Comment:
What does the `__new__` do? Have you checked that it works?

Reply:
I have checked that it works. This subclass of `structseq` appears to be unique, in that (presumably because it creates a tuple of length 1), the argument passed to the constructor does _not_ need to be an iterable. As such, I override the `structseq` constructor here.